lib/assets/javascripts/turbograft/page.coffee

```diff
@@ -19,9 +19,12 @@ Page.refresh = (options = {}, callback) ->
   if options.response
     onlyKeys   = options.onlyKeys   || []
     exceptKeys = options.exceptKeys || []
-    Turbolinks.loadPage null, options.response, true, callback, onlyKeys, exceptKeys
+    Turbolinks.loadPage null, options.response, true, true, callback, onlyKeys, exceptKeys
```

Comment:
Side note: coffeescript can kind of fake named params with a hash:
```coffeescript
func = ({foo, bar}) ->
```
which becomes
```javascript
  func = function(_arg) {
    var bar, foo;
    foo = _arg.foo, bar = _arg.bar;
  };
```
Not good for doing default args for them though :/